packages/common/tsconfig.json

```diff
@@ -2,7 +2,6 @@
     "compilerOptions": {
         /* Visit https://aka.ms/tsconfig.json to read more about this file */
 
-        "composite": true,
```

Comment:
why removing this ?

Reply:
it's a duplicate! further down it's mentioned again